Files/Helpers/UIFilesystemHelpers.cs

```diff
@@ -115,6 +116,12 @@ public static async Task CopyItem(IShellPage associatedInstance)
             {
                 foreach (ListedItem listedItem in associatedInstance.SlimContentPage.SelectedItems)
                 {
+                    if (listedItem is FtpItem ftpItem)
+                    {
+                        items.Add(await new FtpStorageFile(associatedInstance.FilesystemViewModel, ftpItem).ToStorageFileAsync());
```

Comment:
`ToStorageFileAsync` here to use `StorageFile.CreateStreamedFileAsync` for creating a virtual `StorageFile`, but neither pasting in Files (resulting `InvalidCastException`) nor pasting in explorer (resulting "interface is not supported") works.
Any idea to this?

Reply:
Look [here](https://socialeboladev.wordpress.com/2013/03/15/how-to-use-createstreamedfileasync/). Apparently you're supposted to call `request.Dispose()`

Reply:
Confirmed that's not `request.Dispose()`.

Reply:
I think it's a bug related to .NET. 
I'm using C++/WinRT and everything works like a charm:

Foo.cpp:

```cpp
#define UNICODE
#include <iostream>
#include <winrt/Windows.Foundation.Collections.h>
#include <winrt/windows.storage.h>
#include <winrt/windows.storage.streams.h>

using namespace winrt::Windows;
using namespace Storage;
using namespace Storage::Streams;
using namespace Foundation;

void request(const StreamedFileDataRequest request)
{
    auto writer = DataWriter();
    writer.WriteBytes({'h', 'e', 'l', 'l', 'o'});
    auto buffer = writer.DetachBuffer();
    request.WriteAsync(buffer);
}

IAsyncAction create()
{
    auto file = co_await StorageFile::CreateStreamedFileAsync(L"test.txt", &request, nullptr);
    auto folder = co_await StorageFolder::GetFolderFromPathAsync(L"path/to/directory");
    co_await file.CopyAsync(folder, L"test.txt");
}

int main()
{
    winrt::init_apartment();
    create().get();
    winrt::uninit_apartment();
}
```

Build with msvc (or clang):
```
cl Foo.cpp OneCore.lib /std:c++latest /EHsc /await /FeFoo.exe
```

Reply:
We used to have a c++ component till recently I think, we could bring it back if it solves this issue?

Reply:
I create a new UWP project in C# from scratch, and it works again.

Reply:
Resolved in 275c732